scripts/build_utils.sh

```diff
@@ -953,3 +953,11 @@ get_nr_build_jobs() {
     fi
     echo $n_build_jobs
 }
+
+maybe_downgrade_gcc() {
```

Comment:
Would you mind adding a comment to https://bugzilla.redhat.com/show_bug.cgi?id=1726630? Without seeing the Git history here, it's quite hard to understand why we would ever do this.

Reply:
sure. updated and repushed.